test/pythonml/test_context.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import os
+import re
+import unittest
+from unittest import mock
+
+
+class TestContext(unittest.TestCase):
+    """
+    Unit tests for the methods in the Context class defined in the settings.py module
+    """
+
+    def setUp(self):
+        with open('fixtures/settings.py', 'r') as file:
+            unmodified_settings_file = file.readlines()
+        with open('settings.py', 'w') as file:
+            for line in unmodified_settings_file:
+                line = re.sub("PROBLEM_CATEGORY_HERE", 'regression', line)
+                file.write(line)
+        import settings
+        self.context = settings.Context()
+        self.context.context_paths.update({self.context._context_file: 'file'})
+
+    def tearDown(self):
+        os.system('rm -rf .job_context')
+        os.system('rm settings.py')
```

Comment:
Prefer `subprocess.call` over `os.system` 

docs: https://docs.python.org/3/library/subprocess.html#subprocess.call

Example would be `subprocess.call(['rm', '-rf', '.jobcontext'])`